db/migrate/20220915123925_change_user_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    remove_reference :comments, :user, null: false, foreign_key: true
+    add_reference :comments, :author, null: false, foreign_key: { to_table: 'users' }
+  end
```

Comment:
- [ ] Same here, please change user to author and delete foreign_key:

```
  def change
    remove_reference :comments, :author
    add_reference :comments, :author, null: false, foreign_key: { to_table: 'users' }
  end
```

Reply:
I have modified it 👍